src/App.js

```diff
@@ -80,32 +81,39 @@ const App = (props) => {
         } catch (e) {
             chrome.hideGlobalFilter();
         }
+
+        setHasSystemsReady(true);
     }
 
     useEffect(() => {
         initChrome();
```

Comment:
nit-pick: it's an almost redundant function and also named badly (given how much logic we have contained within it not really related to initialising chrome). We would need to refactor it in the soonest future ideally.